### ISSUE TITLE
Put all required software versions into one constants/config file

### ISSUE THREAD

**Issue by james-garriss** [cisagov/ScubaGear#1140]
# 💡 Summary #

If admins want to know what versions that of PowerShell modules and OPA are required, they have to dig through code.  These values should be put into a single file (along with other values) that developers can use and admins can view.

This is part of epic #1136

## Motivation and context ##

This type of information should be found in one file that everyone can see and use.

## Implementation notes ##

- Select one type of file format (perhaps YAML) where dependencies are stored.
- Coordinate with issues #869 and #1088 so that there's one solution for everything
- Implement the solution
- Mod the code to use this constants file
- Update the readme

## Acceptance criteria ##

How do we know when this work is done?

- [ ] There's a single place to find all constants

**Comment by tkol2022** [comment]
@mitchelbaker-cisa @DickTracyII I don't think this issue is necessary anymore because we have the following cmdlets that take care of dependency management. If you agree, close this issue out:

- Initialize-Scuba
- Test-ScubaGearVersion
- Reset-ScubaGearDependencies

**Comment by mitchelbaker-cisa** [comment]
> [@mitchelbaker-cisa](https://github.com/mitchelbaker-cisa) [@DickTracyII](https://github.com/DickTracyII) I don't think this issue is necessary anymore because we have the following cmdlets that take care of dependency management. If you agree, close this issue out:
> 
> * Initialize-Scuba
> * Test-ScubaGearVersion
> * Reset-ScubaGearDependencies

@DickTracyII I believe your new `Get-ScubaGearDependencyStatus` function outputs all required dependency versions if any mismatches occur, which solves the issue of visually outputting the required dependency versions.

I think the other part this issue was getting at though is centralizing PowerShell dependencies (what's in RequiredVersions.ps1 now, which will eventually only be powershell-yaml and Microsoft.Graph.Authentication) and the OPA version. Since the latest OPAVersion is already declared in ScubaConfigDefaults.json, does it make sense to move the list of required PowerShell dependencies into ScubaConfigDefaults.json? Would consider this a low priority refactor since RequiredVersions.ps1 works fine as is.
